### PR TITLE
Stack Component Updates

### PR DESCRIPTION
**Pull request by JoshBowdenConcepts**
## Summary

Some minor updates to the Stack component to make it easier to use and easier to see the documentation.

## List of notable changes:

- **added** custom class option to Stack component
- **updated** placeholder color to be more visible on different screens

## What should reviewers focus on?

- Seeing if the class works for adding custom CSS
- Seeing if the new color appears more readily on their browser

## Steps to test:

1. Open the Stack component in CI-deployed preview environment
2. Go to Stack story in Storybook
3. Verify that Stack behaves as described in the following issue.

## Contributor checklist:

- [ ] All new and existing CI checks pass
- [ ] Tests prove that the feature works and covers both happy and unhappy paths
- [ ] Any drop in coverage, breaking changes or regressions have been documented above
- [ ] New visual snapshots have been generated / updated for any UI changes
- [ ] All developer debugging and non-functional logging has been removed
- [ ] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change

## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

image

 </td>
<td valign="top">

image

</td>
</tr>
</table>

### CODE REVIEW

File: apps/docs/content/components/Stack.mdx [hunk 1]
```diff
@@ -150,15 +150,15 @@ E.g. Providing only `narrow` will apply that value to all larger breakpoints, bu
   }}
 >
   <img
-    src="https://via.placeholder.com/150x75/f5f5f5/f5f5f5.png"
+    src="https://via.placeholder.com/150x75/D0D7DE/D0D7DE.png"
```
Comment (rezrah) [inline apps/docs/content/components/Stack.mdx:153]: Is `D0D7DE` a position on our greyscale? Not a big deal if not, but would be nice to make it that so it's not arbitrary.

File: .changeset/itchy-cycles-relax.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react-brand": patch
+---
+
+Stack component: Added custom class option
```
Comment (rezrah) [inline .changeset/itchy-cycles-relax.md:5]: Suggestion on adding more context..

```suggestion
`Stack` component: Added `className` prop forwarding to the `Stack` root 
```

**Review by rezrah — APPROVED** [review]
Looks good. Thanks @JoshBowdenConcepts :shipit:

File: apps/docs/content/components/Stack.mdx (hunk 1 above)
Comment (JoshBowdenConcepts) [reply apps/docs/content/components/Stack.mdx:153]: It is or at least I got it from the colors page in our Figma file

**Pull request merged**